From c2096b06d462ea91a16e0ed1a184843e21aa98bd Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Wed, 8 Apr 2015 10:35:12 +0800
Subject: [PATCH 469/602] MA-6472 Correct the lightsensor node for libsensor

Libsensor will operate the lightsensor node file
to configurate the device. When we upgrade to 3.14
the node has changed due the DTS changed.
So correct it.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 sabreauto_6q/init.rc  | 2 +-
 sabreauto_6sx/init.rc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index aab96a2..559516a 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -93,7 +93,7 @@ on boot
     chmod 0660 /sys/class/backlight/pwm-backlight.0/brightness
 
     # Set light sensor sysfs path and light sensor threshold lux value
-    setprop ro.hardware.lightsensor "/sys/class/i2c-dev/i2c-1/device/1-0044/"
+    setprop ro.hardware.lightsensor "/sys/class/i2c-dev/i2c-2/device/2-0044/"
     setprop ro.lightsensor.threshold  20
 
     # No bluetooth hardware present
diff --git a/sabreauto_6sx/init.rc b/sabreauto_6sx/init.rc
index 335669e..f11e5af 100755
--- a/sabreauto_6sx/init.rc
+++ b/sabreauto_6sx/init.rc
@@ -93,7 +93,7 @@ on boot
     chmod 0660 /sys/class/backlight/pwm-backlight.0/brightness
 
     # Set light sensor sysfs path and light sensor threshold lux value
-    setprop ro.hardware.lightsensor "/sys/class/i2c-dev/i2c-1/device/1-0044/"
+    setprop ro.hardware.lightsensor "/sys/class/i2c-dev/i2c-2/device/2-0044/"
     setprop ro.lightsensor.threshold  20
 
     # No bluetooth hardware present
-- 
1.8.0

